### DISCUSSION TITLE
Transfer Servers

### DISCUSSION THREAD

**Discussion by ImKomplex** [gameserverapp/Platform#265] [Suggestions]
The Ability to transfer Dediconnect servers from machine x to machine y

**Comment by furio85** [comment]
You can do this.  But I am assuming you want a more "streamlined" way of just clicking and letting it install?  If you don't know.  I have done this by simply, cloning the game server.  Then let it reinstall on the new machine, and FTP and copy the save folder to the new machine.  Granted, for ark this works.  Not sure about the other game servers.

**Comment by mvaessen** [comment]
This was implemented in the previous release
